backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -1844,7 +1844,18 @@ public static VersioningEntry getVersioningEntryFromList(
     return versioningEntry.build();
   }
 
-  public static void validateEntityIdInPredicates(
+  /**
+   * Validating predicate with following condition * Validate if current user has access to the
+   * entity or not where predicate key has an id with only allowed 'EQ' operators * Validate
+   * predicate id is in accessible ids or not * Workspace & workspace type should not allowed in the
```

Comment:
```suggestion
   * Validate predicate id is in accessible ids or not.
   *  Workspace & workspace type should not be allowed in predicates, since they are treated as separate parameters in the find calls.
```